packages/ui/components/ui/tx/view/address-view.tsx

```diff
@@ -24,17 +24,19 @@ export const AddressViewComponent = ({
 
   const accountIndex =
     view.addressView.case === 'visible' ? view.addressView.value.index?.account : undefined;
-  const isRandomized =
+  const isOneTimeAddress =
```

Comment:
Changed to `isOneTimeAddress` since it makes the reassignment of `copyable` on line 32 below make more sense.